translations/combat.tex/cs.po

```diff
@@ -527,11 +527,11 @@ msgid ""
 "The previously played Attack and Defense cards no longer have any effect.}\n"
 "\n"
 msgstr ""
-"\\textit{Po zahrání všech karet pro útok je hozena Kostka Útoku, aby dále upravila množství zranění, které útočící Jednotka způsobí.\n"
+"\\textit{Po zahrání všech karet je pro útok hozena Kostka Útoku, aby dále upravila množství poškození, které útočící Jednotka způsobí.\n"
```

Comment:
> Po zahrání všech karet pro útok je hozena Kostka Útoku
has different meaning
> Po zahrání všech karet je pro útok hozena Kostka Útoku

The original is
> After all cards for the attack have been played, the Attack Die is thrown

Reply:
You are right.